paddle/api/paddle_ld_flags.py

```diff
@@ -56,9 +69,9 @@ def parent_dir_str(self):
 
         def libs_str(self):
             libs = [
-                "-Wl,--whole-archive",
+                whole_start,
```

Comment:
All already done this via "-Wl, all_load" in line 48, /Users/liaogang/baidu/Paddle/paddle/setup.py.in.